docs/content/new-tutorial/your-first-asset.mdx

```diff
@@ -0,0 +1,187 @@
+- gets the titles of the top 100 stories
+- makes a chart with this data and save it
+
+You want this visualization updated daily, and you want to use Dagster to orchestrate it. The first step is to get the ingestion moved over to Dagster.
+
+---
+
+## Writing your first asset
+
+Let's start moving that script over to Dagster and build your first asset while doing so!
+
+In your scaffolded Dagster project, there will be a directory called `tutorial-project`. If you named your project differently when running the Dagster CLI, the directory will be named after that name.
+
+In the `tutorial-project` directory, you will see a file called `assets.py`. This is where you will put your first data pipeline.
+
+### Ingesting data
+
+To get started, you will fetch data from the Hacker News API. You can do this by adding the following code snippet from to `assets.py`:
+
+```python
+import requests
+
+newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
+top_100_newstories = requests.get(newstories_url).json()[:100]
+```
+
+This is your first asset. It is a list of integers representing the IDs for the current top stories on Hacker News.
+
+Next, let's turn it into a function:
+
+```python
+import requests
+
+def hackernews_topstory_ids():
+    newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
+    top_100_newstories = requests.get(newstories_url).json()[:100]
+    return top_100_newstories
+```
+
+By turning it into a function, you've made the first step into making it an asset. Now, you need to add the `@asset` decorator to the function.
+
+```python
+import requests
+from dagster import asset
+
+@asset
+def hackernews_topstory_ids():
+    newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
+    top_100_newstories = requests.get(newstories_url).json()[:100]
+    return top_100_newstories
+```
+
+That's all it takes to get started. Dagster now knows that this is an asset. In future sections, you'll see how you can add more metadata, schedule when to refresh the asset, and more.
+
+And now you're done! Time to go into the Dagster UI and see what you've built.
+
+### Exploring the Dagster UI
+
+Dagster has a UI called Dagit. It's a great way to explore your data pipelines and see what's going on.
+
+To launch Dagit, set your current directory to `/tutorial-project`, and run the following command in your terminal:
+
+```bash
+dagit
+```
+
+If `localhost:3000` is not already being used, Dagit will use that port. Navigate to `localhost:3000` in your browser to see the Dagit UI.
+
+You should see a screen that looks like this:
+
+<Image
+alt="the Dagit asset graph with no materializations"
+src="/images/tutorial/dagit-default.png"
+width={1392}
+height={1167}
+/>
+
+You can see that Dagit has detected your `hackernews_topstory_ids` asset. The UI isn't very useful yet because the asset hasn't been materialized yet. Let's do that next.
+
+---
+
+## Materializing your asset
```

Comment:
I think it's important to communicate that Dagit isn't the only way to materialize your asset. Part of what's so cool about Dagster is that it has really lightweight ways of running stuff - e.g. on the CLI and with the Python APIs.